scripts/make_tiny_4mula_index.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import json
+import os
+from mirdata.validate import md5
+from mirdata.download_utils import RemoteFileMetadata, download_from_remote
+from numpy import save
+import pyarrow.parquet as pq
```

Comment:
`pyarrow` is not a dependency in `mirdata`, is it possible to load the data with `csv` instead?

Consider this comment is low priority because `make_index` scripts are here for reproducibility and is rare that users run them

Reply:
But it would be good practice :)

Reply:
The `parquet` format was 300% better than `csv` to compress the data. I think that parquet is a format interesting to reduce the use of disk and memory. I used `pyarrow`  due to the feature to read the dataset in batches to reduce memory use.